integration/integration_test.go

```diff
@@ -80,6 +81,8 @@ func TestMain(m *testing.M) {
 		os.Setenv("GADGET_IMAGE_FLAG", "--image "+*image)
 	}
 
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
As we discussed, can you please add a print to this seed to be able to reproduce the CI in case of bug within it caused by a particular seed?

Reply:
Done.